tests/test_federal_senate_dataset.py

```diff
@@ -0,0 +1,34 @@
+import os
+from tempfile import gettempdir
+from unittest import main, skipIf, TestCase, TestLoader
+
+from serenata_toolbox.federal_senate.federal_senate_dataset import FederalSenateDataset
+
+if os.environ.get('RUN_INTEGRATION_TESTS') == '1':
+    TestLoader.sortTestMethodsUsing = None
+
+class TestFederalSenateDataset(TestCase):
+    def setUp(self):
+        self.path = gettempdir()
+        self.subject = FederalSenateDataset(self.path)
+
+    @skipIf(os.environ.get('RUN_INTEGRATION_TESTS') != '1',
+            'Skipping integration test')
+    def test_fetch_saves_raw_files(self):
+        self.subject.fetch()
+        names = ['federal-senate-{}.csv'.format(year) for year in range(2008, 2018)]
+        for name in names:
+            file_path = os.path.join(self.path, name)
+            assert(os.path.exists(file_path))
```

Comment:
Let's say your test creates a bizarre file: `/tmp/my-bizarre-and-beloved-test-side-effect.pkl`:

```python
class TestSomething(TestCase):

    def setUp(self):
        self.path = gettempdir()
        self.file_path = os.path.join(self.path, 'my-bizarre-and-beloved-test-side-effect.pkl')
        …

    def tearDowm(self):
        os.remove(self.file_path)

    def test_something(self):
        pass  # something that creates the bizarre file
```

Everything on `setUp` runs before every test. Everything in `tearDown` runs after every test. So you can be sure that after every test the bizarre file is deleted. Surely it might need an is `os.path.exist`, or a `try`/`except` if the file wasn't created in every test… but this is the general idea.

Makes sense?